step-release-vis/src/app/models/SnapshotInterval.ts

```diff
@@ -0,0 +1,11 @@
+import {Snapshot} from '../models/Data';
+
+/*
+  Used for describing which positions of the mouse corelate to which snapshot to be displayed.
+  If the mouse is in [start, end] on the X axis, show details about this snapshot in the tooltip.
+*/
+export class SnapshotInterval {
+  start: number;
```

Comment:
what is the meaning of the value?
is this in pixels?
were is 0? (screen border, graph border, etc...
what is the max?


what will be the star for a given snapshot? half point between the last and this?

Reply:
done